src/constants.ts

```diff
@@ -21,7 +21,7 @@ export const MAX_BLOCKS_HISTORY = 7;
 
 export const SETUP_RETRY_TIMEOUT = 20 * 1000; // 20s
 
-export const FETCH_POOL_IDENTIFIER_TIMEOUT = 1 * 1000; // 1s
+export const FETCH_POOL_IDENTIFIER_TIMEOUT = 60 * 1000; // 1s // TODO change once finished testing
```

Comment:
I will leave this comment as `request changes`, so we won't miss this on merging